operations/payments.md

```diff
@@ -401,6 +401,59 @@ Returns all payments in the system, filtered by various parameters. At least one
 | `Payments` | array of [Payment](#payment) | required | The list of filtered payments. |
 | `Cursor` | string | required | Unique identifier of the last and hence oldest payment returned. This can be used in [Limitation](../guidelines/pagination.md#limitation) in a subsequent request to fetch the next batch of payments. |
 
+## Refund payment
+
+Refunds a specified payment.
```

Comment:
Sounds good, maybe we could explicitly mention here that only creditCards or alternative payments can be refunded and if partner needs more details then provided in the payments/refund response, they can call the payments/getAll with the refundId. Also, I added Kind field to the payments/getAll so we can add it to the documentation in this PR too, please.